src/Storefront/Page/Account/Login/AccountLoginPageLoader.php

```diff
@@ -95,7 +95,7 @@ private function getCountries(SalesChannelContext $salesChannelContext): Country
     {
         $criteria = (new Criteria())
             ->addFilter(new EqualsFilter('active', true))
-            ->addAssociation('country.states');
```

Comment:
can you add a testcase that verifies that the countries are loaded with the states association?

Reply:
I don't have the test environment set up, so if somebody else has time for that...